### PR TITLE
UP-4954:  CAS AuthN -- Prevent uPortal from requiring a CAS PGT by de…

### PR DESCRIPTION
**Pull request by drewwills**
…fault

https://issues.jasig.org/browse/UP-4954

##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]

##### Description of change

Most uPortals in 2017 don't use PGTs.

If you need it, turn the feature on;  but the default should be that a PGT is not required from CAS.

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Comment by drewwills** [comment]
@asgrant and @bpowell -- can the two of you confirm that this change exhibits the behavior you're looking for?

File: uPortal-webapp/src/main/resources/properties/security.properties [hunk 1]
```diff
@@ -46,12 +46,17 @@ cas.protocol=http
...
 cas.ticketValidationFilter.service=${portal.protocol}://${portal.server}${portal.context}/Login
-cas.ticketValidationFilter.proxyReceptorUrl=/CasProxyServlet
 cas.ticketValidationFilter.ticketValidator.server=${cas.protocol}://${cas.server}${cas.context}
-cas.ticketValidationFilter.ticketValidator.proxyCallbackUrl=${portal.protocol}://${portal.server}${portal.context}/CasProxyServlet
 org.apereo.portal.security.provider.cas.CasAssertionSecurityContextFactory.enabled=false
 org.apereo.portal.security.provider.cas.CasAssertionSecurityContextFactory.credentialToken=ticket
 
+## If the following 2 properties are defined (either with the values shown or other valid values),
+## CAS authentication will also request a Proxy-Granting Ticket (PGT).  This feature requires
+## special permission in the CAS Servicec Manager.
```
Comment (asgrant) [inline uPortal-webapp/src/main/resources/properties/security.properties]: Should this be CAS "Service" Manager?

Comment (drewwills) [reply uPortal-webapp/src/main/resources/properties/security.properties]: Noted.  I'll tackle that.

**Comment by drewwills** [comment]
Merge of this PR is pending some testing at OU.

**Comment by asgrant** [comment]
Drew I tested with our uPortal 4 and 5 versions of MySAIL and I could confirm in both cases the log file did not have the PGT Callback once I added this modification. Clicking through the application worked for me as well, nothing unexpected. I also tested this with our old CAS 4.x server and the CAS protocol in Shibboleth 3.3.1. Everything looks good.

**Comment by drewwills** [comment]
Thank you so much @asgrant and OU!

**Pull request merged**